ci.yaml

```
@@ -0,0 +1,51 @@
+variables:
+  # Avoid expensive initialization of dotnet cli, see: https://donovanbrown.com/post/Stop-wasting-time-during-NET-Core-builds
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+
+stages:
+  - stage: Build
+    jobs:
+      - job: 'Full_Build'
+        pool:
+          vmImage: windows-latest
+        steps:
+        - pwsh: |
+            Import-Module .\tools\appveyor.psm1
+            Invoke-AppveyorInstall -SkipPesterInstallation
+            ./build.ps1 -Configuration 'Release' -All
+            ./PSCompatibilityCollector/build.ps1 -Configuration 'Release'
```

```suggestion
            Import-Module .\tools\appveyor.psm1
            Invoke-AppveyorInstall -SkipPesterInstallation
            ./build.ps1 -Configuration 'Release' -All
            ./PSCompatibilityCollector/build.ps1 -Configuration 'Release'
```

./build.ps1 -package -publish